api/app/scripts/audit_knowledge_update_candidates.py

```
@@ -0,0 +1,670 @@
+    )
+
+    print(f"Wrote audit artifacts to {out_dir}")
+    print(json.dumps(summary, indent=2, sort_keys=True))
+
+
+def _parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument(
+        "--export",
+        type=Path,
+        default=Path("/tmp/bisq_support_production_export_latest.json"),
```

Ruff (0.15.17)

[error] Probable insecure usage of temporary file or directory: "/tmp/bisq_support_production_export_latest.json"

(S108)